### ISSUE TITLE
Dead personnel are being updated by rewards auto-assignment

### ISSUE THREAD

**Issue by OrbMonky** [MegaMek/mekhq#4784]
### Environment
Windows 10 / Java 17 / 0.50.00 release

### Description
![image](https://github.com/user-attachments/assets/9509a247-7cdf-4468-b653-533136c71d63)
Dead are given awards from campaigns that started after they were deceased

### Files
[AWARDS.zip](https://github.com/user-attachments/files/16881331/AWARDS.zip)
[customs.zip](https://github.com/user-attachments/files/16881332/customs.zip)

**Comment by IllianiCBT** [comment]
I’m assuming by ‘campaigns’ you mean contracts? If you turn off ‘posthumous awards’ that’ll work around this issue until I have time to fix it during 50.01’s dev cycle :)

**Comment by OrbMonky** [comment]
Yes, sorry, contract. I'll turn it off since it doesn't account for contracts that the deceased wasn't a part of. Thanks!

**Comment by IllianiCBT** [comment]
Closing as resolved
